test/targets/babel/babel.test.js

```diff
@@ -0,0 +1,55 @@
+/* global describe, it, after, before */
+'use strict';
+
+var fs = require('fs');
+
+var should = require('should');
+
+var axon = require('axon');
+var requester = axon.socket('req');
+
+var server = require('../../../lib/server');
+
+describe('Babel', function () {
+
+  before(function () {
+    server.start();
+    requester.connect('tcp://localhost:5555');
+  });
+
+  it('Should process valid ES6 and pass back the compiled source', function (done) {
+    fs.readFile(__dirname + '/sample.js', function (error, file) {
+      requester.send({
+        language: 'es6',
+        source: file.toString()
+      }, function (res) {
+        (res.error === null).should.be.true;
+        (res.output.errors === null).should.be.true;
+        res.output.result.should.exist;
+        done();
+      });
+    });
+  });
+
+  it('Should process invalid ES6 and give back an error', function (done) {
+    fs.readFile(__dirname + '/broken.js', function (error, file) {
+      requester.send({
+        language: 'es6',
```

Comment:
Good question...it's been a while since I added a new processor. I *think* it needs to match the processor name, but I'll test the branch locally to confirm. I've got some time today that I should be able to finish the merge and push.